db/migrate/20150428031431_add_user_id_to_mentor.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserIdToMentor < ActiveRecord::Migration
```

Comment:
There should only be one migration per PR.  please add this column to the above migration